templates/docker-compose.template

```diff
@@ -48,15 +45,12 @@ services:
       - {{I2C_DEVICE}}:{{I2C_DEVICE}}
     restart: on-failure
     environment:
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
-      - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - I2C_DEVICE={{I2C_DEVICE}}
```

Comment:
Why do we need this i2c device env variable?

Reply:
to detect ecc in the current startup script. Will be removed when convert startup to python and start using variant information. https://github.com/NebraLtd/hm-gatewayrs/issues/24

Reply:
Ah ok fair enough